app/graphql/types/movie_type.rb

```diff
@@ -15,6 +15,9 @@ class MovieType < Base::Object
     field :budget, Integer, null: true, description: I18n.t("#{I18N_PATH}.fields.budget")
     field :runtime, Integer, null: true, description: I18n.t("#{I18N_PATH}.fields.runtime")
     field :original_language, String, null: true, description: I18n.t("#{I18N_PATH}.fields.original_language")
+    field :is_favorite, Boolean, null: false, description: I18n.t("#{I18N_PATH}.fields.is_favorite")
+    field :is_watchlist, Boolean, null: false, description: I18n.t("#{I18N_PATH}.fields.is_watchlist")
+    field :removed_movie_id, String, null: false, description: I18n.t("#{I18N_PATH}.fields.removed_movie_id")
```

Comment:
please move this field to the separate type. what if I want to fetch the movie info for the view page? what this field should return?)